src/vs/workbench/services/extensions/browser/webWorkerExtensionHost.ts

```diff
@@ -83,9 +83,10 @@ export class WebWorkerExtensionHost extends Disposable implements IExtensionHost
 		}
 		COI.addSearchParam(suffixSearchParams, true, true);
 
-		const suffix = `?${suffixSearchParams.toString()}`;
+		let suffix = `?${suffixSearchParams.toString()}`;
 
 		const iframeModulePath = 'vs/workbench/services/extensions/worker/webWorkerExtensionHostIframe.html';
+		const locale = localStorage.getItem('vscode.nls.locale') || navigator.language.toLowerCase();
```

Comment:
Since this is running on the main side, I think you can just use `language` or `Language.value()`:
https://github.com/microsoft/vscode/blob/7a104d4d1141d676d71eb78b2dda72a766da5f7c/src/vs/base/common/platform.ts#L182-L188

the implementation of this will grab the local storage value or navigator value.